bouncer.go

```diff
@@ -179,6 +179,12 @@ func validateCreateStruct(errors Errors, obj interface{}) Errors {
 		fieldValue := val.Field(i).Interface()
 		zero := reflect.Zero(field.Type).Interface()
 
+		// If the field Value is a string, then trim the leading spaces
+		fieldActualValue := val.Field(i)
+		if fieldActualValue.Kind() == reflect.String {
+			fieldActualValue.SetString(strings.TrimSpace(fieldValue.(string)))
```

Comment:
When you get a moment, can you explain why you created and use `fieldActualValue` instead of running the trim on say `fieldValue` or `val.Field(i).Interface()`?  I guess I don't see where `fieldActualValue` gets used / checked later.

Reply:
In order to replace the value of a field in the original struct, you need the ``SetString()`` method from ``val.Field(i)``, and such method is not available in ``fieldValue := val.Field(i).Interface()``

Reply:
The readability of code could be improved if we let
```
fieldValue := val.Field(i)
fieldValueInterface := fieldValue.Interface()
```
which will required some refactoring of other codes as well